docs/InstallGuide.md

```diff
@@ -1,29 +1,24 @@
-# OpenVNet #
 
-# Preliminary Operations and Installation #
+Overview
+--------
 
-> **Note:**
-This document explains the installation of OpenVNet.  However, it was
-written for an older version of OpenVNet and does not apply
-to the current version.  We plan to update this guide soon.  Until
-then, we are leaving this guide here and hope it will be useful for those
-who wish to learn more about the components that make up OpenVNet.
+Welcome to the world of OpenVNet! With this installation guide we help you
+create a very simple yet innovative virtual network environment.
 
+![vnet_minimum](https://www.dropbox.com/s/dezarv561fg7sdj/vnet_minimum.png?dl=1)
 
-Installation Requirements
--------------------------
+On a given physical server (here named as server1) there is one virtual network whose network address is 10.100.0.0/24 and 2 virtual machines joining it.
 
-### System Requirements
 
-+ RHEL 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
+Requirements
+------------
 
 ### Network Requirements
 
 + Local Area Network (LAN)
 + Internet connection
 
-Pre-setup
+Installation
 ---------
 
 Download the openvnet.repo file and put it to your /etc/yum.repos.d/ directory.
```

Comment:
A brief (single sentence) explanation of what is in the openvnet and openvnet-third-party repositories would be nice.